microsoft-edge/webview2/concepts/end-user-faq.md

```diff
@@ -0,0 +1,70 @@
+---
+title: WebView2 end-user FAQ
+description: Information for end users about WebView2 processes, memory and CPU usage, and automatic installation.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 03/30/2023
+---
+# WebView2 end-user FAQ
+
+This FAQ is for end users, to explain the `msedgewebview2.exe` application on your computer.  If your question isn't listed here, you can click the **Feedback** button in the top-right corner, to enter a question as an [Issue in the edge-developer Docs repo](https://github.com/MicrosoftDocs/edge-developer/issues), and your question might be added to this article.  See also [Contact the WebView2 Team](../contact.md).
+
+
+<!-- ====================================================================== -->
+## What is WebView2?
+
+WebView2 is a way for app developers to embed web content (such as HTML, JavaScript, and CSS) in Windows applications. By including the WebView2 control in an app, a developer can write code for a website or web app, and then reuse that web code in their Windows application, saving time and effort.  See [Introduction to Microsoft Edge WebView2](../index.md).
+
+
+<!-- ====================================================================== -->
+## What's the purpose of the WebView2 executable?
```

Comment:
Corp-level style guide is flexible: per Dev of orig. PR, the audience for this article is power users, ie technical audience, thus 'executable' is allowed by the corp-level style guide:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/executable-exe - Use _executable_ file only in content for a technical audience. For a general audience, use _program file_ instead.